website/scripts/build-static-content.js

```diff
@@ -93,28 +94,51 @@ module.exports = {
           return list;
         }, []);
 
-        // Talk to GitHub and get additional information about each contributor.
         let githubDataByUsername = {};
-        await sails.helpers.flow.simultaneouslyForEach(githubUsernames, async(username)=>{
-          githubDataByUsername[username] = await sails.helpers.http.get.with({
-            url: 'https://api.github.com/users/' + encodeURIComponent(username),
-            headers: { 'User-Agent': 'Fleet-Standard-Query-Library', Accept: 'application/vnd.github.v3+json' }
-          });
-        });//∞
-
-        // Now expand queries with relevant profile data for the contributors.
-        for (let query of queries) {
-          let usernames = query.contributors.split(',');
-          let contributorProfiles = [];
-          for (let username of usernames) {
-            contributorProfiles.push({
-              name: githubDataByUsername[username].name,
-              handle: githubDataByUsername[username].login,
-              avatarUrl: githubDataByUsername[username].avatar_url,
-              htmlUrl: githubDataByUsername[username].html_url,
+
+        if(local) {// If the local flag was provided, we'll skip querying GitHubs API
+          sails.log('Local run: Skipping GitHub API requests for contributer profiles.\nNOTE: The names of contributors in the standard query library will be set to the contributors GitHub username instead of the name on their profile. To see how the standard query library will look on fleetdm.com, run this script without the `--local` flag.');
+          // Because we're not querying GitHub to get the real names for contributer profiles, we'll use their GitHub username as their name and their handle
+          for (let query of queries) {
+            let usernames = query.contributors.split(',');
+            let contributorProfiles = [];
+            for (let username of usernames) {
+              contributorProfiles.push({
+                name: username,
+                handle: username,
+                avatarUrl: 'https://github.com/'+username+'.png?size=200',
```

Comment:
Better to be explicit that this is fake, so that when developing, it's clear.

A good option is https://placekitten.com

i.e.
```
avatarUrl: 'https://placekitten.com/200/200',
```

Reply:
That URL for the avatarUrl is not fake, `https://github.com/{username}.png` redirects to the GitHub profile picture for that user.

Do you think it would be better to use a placeholder to be explicit that the site was built with missing contributor data?

Reply:
Yes, make it explicit that it is fake.